packages/node/src/configure/configure.module.ts

```diff
@@ -42,7 +44,33 @@ function defaultSubqueryName(config: Partial<IConfig>): MinConfig {
   } as MinConfig;
 }
 
-const logger = getLogger('configure');
+// Check if a subquery name is a valid schema name
+function validDbSchemaName(name: string): boolean {
```

Comment:
schema name is case insensitive, `toLowerCase()` before continue

Reply:
done